podman/parameters/defaults.yaml

```diff
@@ -87,6 +87,7 @@ values:
   debian_experimental: false
   debian_unstable: false
   install_method: pkg
+  python_install_method: pip
```

Comment:
Just a note:

New parameters would usually need to go into `.cruft.json` first (which manages much of the boilerplate). I will do this myself since the `cruft` setup is currently a bit unfriendly to contributors with requiring custom patches.

I might need to look into switching to `copier` to make this smoother.

Reply:
Interesting, I haven't used it before. Thanks for taking care of it!